microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -57,6 +57,8 @@ def ltp_lite(
         # parse variables
         tests = variables.get("ltp_test", "")
         skip_tests = variables.get("ltp_skip_test", "")
+        use_block_drive = variables.get("use_block_drive", True)
```

Comment:
the name should be `use_os_drive` like in the code, and default value is "False".

Reply:
Our tests run with the default value of `True`. Few tests in LTP lite require this to be set and this increases the number of tests which are covered by LTP lite

Reply:
Also, OS disk is different from Block drive. Using block drive means LTP can use any block drive

Reply:
The os drive is a block device too. The variable name is confusing here.

Reply:
Fixed!